hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/om-overview.html

```diff
@@ -49,6 +45,36 @@ <h2>Status</h2>
     </tbody>
 </table>
 
+<h4 ng-show="$ctrl.overview.jmx.RatisRoles.length == 1 && $ctrl.overview.jmx.RatisRoles[0].length == 1">Exception: {{$ctrl.overview.jmx.RatisRoles[0]}}</h4>
+<div ng-show="$ctrl.overview.jmx.RatisRoles.length > 1">
+    <h2>OM Roles (HA)</h2>
+    <table class="table table-striped table-bordered" class="col-md-6">
+        <thead>
+        <tr>
+            <th>Host Name</th>
+            <th>Node ID</th>
+            <th>Ratis Port</th>
+            <th>Role</th>
+        </tr>
+        </thead>
+        <tbody ng-repeat="roles in $ctrl.overview.jmx.RatisRoles">
+        <tr class="om-roles-background" ng-if="$ctrl.role.Id == roles[1]">
+            <td>{{roles[0]}}</td>
```

Comment:
If there is exception, how the output looks? do still accessing roles[1],... can have issue over UI in exception case?

Reply:
Could you help answering this question @smitajoshi12?

Reply:
ok, just display exception message below "OM Roles (HA)" heading